spec/models/comment_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  describe 'Validations for the Comment model' do
+    let(:comment) { build(:comment) }
+
+    it 'requires the presence of text' do
+      comment.text = nil
+      expect(comment).to be_invalid
+    end
+
+    it 'requires the presence of post_id' do
+      comment.post_id = nil
+      expect(comment).to be_invalid
+    end
```

Comment:
- [ ] Kindly create some positive test also, ex:- if comment.text = "something", it should be valid